GoogleForCreators/web-stories-wp
# Pre-Publish Checklist: Enable Pre-Publish Checklist.

## Summary

Removes the feature flag and enables the pre-publish checklist.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

1. The strings noted in this document can be used as a guideline to behaviour and expected messages presented to the user - https://docs.google.com/spreadsheets/d/1PjaRp4bueQ3Cv14A7I696m0vjGCIX_kN9tIGh-V-Bkc/edit?pli=1#gid=0
2. Please review all active bugs against this issue first so as to understand where know issues exist - https://github.com/google/web-stories-wp#workspaces/web-stories-5e94d3aced449034e1e9b226/board?labels=bug%20bash%3A%201.2,group%3A%20pre-publish%20panel&notFullScreen=false&repos=235435637&showPRs=false&showClosed=false

---



Fixes #
